app/assets/stylesheets/application.css

```diff
@@ -27,16 +78,86 @@
   border: 1px solid black;
 }
 
-.details {
+.container {
   display: flex;
-  gap: 20px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
 }
 

```

Comment:
- [ ] Good job so far, design is coming up well 👌 Let's give our design a better look with by tweaking the following:
  * The top blue navbar should have a **whitish** text color, and should be **uppercase** and use **san-serif** font type or similar.
  * Let's remove the border around the navbar
  * The big green button should always stay at the bottom of the page, and text should be **white** and **uppercase**.
  * Let's enforce proper spacing and positioning of elements, as seen in the template design.

**Your design:**
**Your splash screen:**        |  **Your Login:**        |  **Your categories page:**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227757282-83fe6d0d-bd24-498c-aa6e-8be18c254e85.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227757349-0d4ce0f3-a24a-4903-a9ee-161ba794c415.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227757230-2e99b678-f6dc-4a86-9e91-e5a59bbd9cd9.png">


**Expected design:**
**Original splash screen:**        |  **Original login screen:**        |  **The original transaction:**     
:-------------------------:|:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/198862103-fe9fe8a8-7a06-4a03-92f6-8528c3d4d2f1.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227751557-e1341b09-89ea-42aa-ad37-cd38d2510b53.png"> | <img width="800" alt="Home Page" src="https://user-images.githubusercontent.com/12745474/227751375-40282971-5bde-49ca-9612-8cc1be58ba99.png">

Reply:
thanks so much, I think the dead line is within 17 min.

Reply:
I have deployed by github also.

Reply:
in my screen the margins are ok.

Reply:
![signup](https://user-images.githubusercontent.com/60324033/227758421-41e9fdae-777e-464f-889f-dc2c7a8939fa.PNG)

Reply:
![transaction](https://user-images.githubusercontent.com/60324033/227758583-fe39db48-2efb-4afa-9778-defc9264d283.PNG)

Reply:
I will deploy by render.